content/guides/monitors.md

```diff
@@ -226,10 +226,68 @@ To schedule downtime, click the "Schedule Downtime" button in the upper right.
+
+### Find the Monitors
+
+Advanced search lets you query monitors by any combination of monitor attributes:
+
+* Title and message body (text search)
+* Status — Alert, Warn, No Data, Ok 
+* Scope — e.g. *, role:master-db
+* Type — metric, integration, apm, etc
+* Muted (or not)
+* Creator — who created the monitor
+* ID
+* Service tags
+* Team tags
+* Env tags
+* Notification target — e.g. you@example.com, slack-ops-oncall
+* Metric or service check monitored — e.g. system.cpu.user, http.can_connect
+
+To run a search, construct your query using the checkboxes on the left and/or the search bar along the top. When you check the boxes, the search bar updates with the equivalent query. Likewise, when you modify the search bar query (or write one from scratch), the checkboxes update to reflect the change. In any case, query results update in real-time as you edit the query; there's no 'Search' button to click.
+
+#### Check the boxes
+
+When you don't need to search monitor titles and bodies for specific text, your search is a quick click or two away. Check as many boxes as you need to find your desired monitors, keeping the following in mind:
+
+* Checking attributes from different fields will AND the values, e.g. `status:Alert type:Metric` (the lack of an operator between the two search terms implies AND)
+* Checking attributes within the same field will often OR the values, e.g. `status:(Alert OR Warn)`, but there are some exceptions: checking multiple scopes or service tags ANDs them.
+* Some fields do not allow you to select multiple values, e.g. when you tick a metric or service check, the other metrics/checks disappear from the list until you untick your selection.
+* The Triggered checkbox under the Status field means `status:(Alert OR Warn OR "No Data")`, not `status:Triggered`. Triggered is not a valid monitor status.
+* The Muted checkbox appears under the Status field, but Muted is actually its own field; checking it adds `muted:true` to your query, not `status:muted`.
+* The Metric/Check field is always called `metric` in the query, e.g. selecting the check `http.can_connect` adds `metric:http.can_connect` to your query.
+
+For fields that have an arbitrary (i.e. large) number of values across all monitors—Service tag, Scope, Metric/Check, Notification—use the field-specific search bars to find the value you're looking for.
+
+When you need to run a more complex search than the checkboxes allow, use the search bar to edit your query or write a new one.
+
+#### Write a query
+
+The most common reason to write a query is to search for specific text across all monitor names (i.e. titles) and message bodies. A simple search of `postgresql` will return all monitors with `postgresql` anywhere in the name or message body. To search on title or message body, but not both, qualify the search term with the field name, e.g. `name:postgresql`.
```

Comment:
@adimusset the column in the UI is still labeled 'NAME'. I suppose we want to change this to 'TITLE'?